tests/modules/commerce_redirect_test/src/Form/RedirectTestForm.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\commerce_redirect_test\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Tests throwing a redirect inside of a form.
+ */
+class RedirectTestForm extends FormBase {
```

Comment:
Now that we only have a single form class (+ routing file), we might as well merge the test module into commerce_test, cleaner that way.